docs/core/project-sdk/msbuild-props-desktop.md

```diff
@@ -0,0 +1,175 @@
+
+You can completely disable all implicit includes by setting the [`EnableDefaultItems` property](msbuild-props.md#enabledefaultitems) to `false`.
+
+## WPF settings
+
+- [UseWPF](#usewpf)
+- [EnableDefaultApplicationDefinition](#enabledefaultapplicationdefinition)
+- [EnableDefaultPageItems](#enabledefaultpageitems)
+
+For more information about standard .NET SDK project settings, see [MSBuild reference for .NET SDK projects](msbuild-props.md).
+
+### UseWPF
+
+The `UseWPF` property controls whether or not to include references to WPF libraries. This also alters the MSBuild pipeline to correctly process a WPF project and related files. The default value is `false`. Set the `UseWPF` property to `true` to enable WPF support. You can only target the Windows platform when this property is enabled.
+
+```xml
+<PropertyGroup>
+  <UseWPF>true</UseWPF>
+</PropertyGroup>
+```
+
+.NET 5.0 projects will automatically import the [.NET Desktop SDK](#enable-net-desktop-sdk) when this property is set to `true`.
+
+.NET Core 3.1 projects need to explicitly target the [.NET Desktop SDK](#enable-net-desktop-sdk) to use this property.
+
+### EnableDefaultApplicationDefinition
+
+The `EnableDefaultApplicationDefinition` property controls whether `ApplicationDefinition` items are implicitly included in the project. The default value is `true`. Set the `EnableDefaultApplicationDefinition` property to `false` to disable the implicit file inclusion.
+
+```xml
+<PropertyGroup>
+  <EnableDefaultApplicationDefinition>false</EnableDefaultApplicationDefinition>
+</PropertyGroup>
+```
+
+This property requires that the [`EnableDefaultItems` property](msbuild-props.md#enabledefaultitems) property is set to `true`, which is the default setting.
+
+### EnableDefaultPageItems
+
+The `EnableDefaultPageItems` property controls whether `Page` items, which are _.xaml_ files, are implicitly included in the project. The default value is `true`. Set the `EnableDefaultPageItems` property to `false` to disable the implicit file inclusion.
+
+```xml
+<PropertyGroup>
+  <EnableDefaultPageItems>false</EnableDefaultPageItems>
+</PropertyGroup>
+```
+
+This property requires that the [`EnableDefaultItems` property](msbuild-props.md#enabledefaultitems) property is set to `true`, which is the default setting.
+
+## WinForms settings
+
+- [UseWindowsForms](#usewindowsforms)
+
+For more information about standard .NET SDK project properties, see [MSBuild reference for .NET SDK projects](msbuild-props.md).
+
+### UseWindowsForms
+
+The `UseWindowsForms` property controls whether or not your application is built to target Windows Forms. This property alters the MSBuild pipeline to correctly process a WinForms project and related files. The default value is `false`. Set the `UseWindowsForms` property to `true` to enable WinForms support. You can only target the Windows platform when this setting is enabled.
```

Comment:
```suggestion
The `UseWindowsForms` property controls whether or not your application is built to target Windows Forms. This property alters the MSBuild pipeline to correctly process a Windows Forms project and related files. The default value is `false`. Set the `UseWindowsForms` property to `true` to enable Windows Forms support. You can only target the Windows platform when this setting is enabled.
```

Reply:
What is the reasoning on this? I'm not sure when to use WinForms vs Windows Forms. I did (at the start of the article) say `Windows Forms (WinForms)`

Reply:
I missed the top bit :)
But as a general practice I prefer to use "Windows Forms", and be consistent throughout - I find it jarring when terms constantly change back and forth. Could be just me though.